lnbits/static/js/index.js

```diff
@@ -101,8 +101,8 @@ window.app = Vue.createApp({
     }
   },
   created() {
-    this.description = SITE_DESCRIPTION
-    this.allowedRegister = ALLOWED_REGISTER
+    this.description = this.SITE_DESCRIPTION
```

Comment:
is `this.SITE_DESCRIPTION` correct (using `this`)?

Reply:
this uses windowMixing instead of the global, i think yes, this is cleaner